src/DenoHTTPWorker.test.ts

```diff
@@ -1,19 +1,54 @@
-import { it as _it, describe, expect } from "vitest";
-import { newDenoHTTPWorker } from "./index.js";
+import { it as _it, beforeAll, describe, expect } from "vitest";
+import { DenoHTTPWorker, newDenoHTTPWorker } from "./index.js";
 import fs from "fs";
 import path from "path";
-import http2 from "http2-wrapper";
 
 // Uncomment this if you want to debug serial test execution
-const it = _it.concurrent;
-// const it =   _it;
+// const it = _it.concurrent;
+const it = _it;
+
+const DEFAULT_HTTP_VAL = `export default async function (req: Request): Promise<Response> {
+  return Response.json({ ok: true })
+} `;
+
+const jsonRequest = (
+  worker: DenoHTTPWorker,
+  url: string,
+  opts?: { headers?: { [key: string]: string }; body?: string }
+): Promise<any> => {
+  return new Promise((resolve, reject) => {
+    let req = worker.request(url, opts || {}, (resp) => {
+      const body: any[] = [];
+      resp.on("error", reject);
+      resp.on("data", (chunk) => {
+        body.push(chunk);
+      });
+      resp.on("end", () => {
+        resolve(JSON.parse(Buffer.concat(body).toString()));
```

Comment:
Mildly evil, but JSON.parse can parse Buffers when you're in Node.js and it might be a small performance boost for big bodies